docker-compose.yml

```diff
@@ -159,20 +159,21 @@ services:
       - .:/app
       - dev-build-cache:/target:overlay
     environment:
-      - LISTEN=[::]:8000
-      - RPC_SERVER_NODES=server-testnet-geth:8545
-      - L2_RPC_URL=http://leader-testnet-geth:8545
-      - L1_RPC_URL=http://l1-testnet-geth:8545
-      - L1_BRIDGE=0x936a70c0b28532aa22240dce21f89a8399d6ac60
-      - L1_PRIV=$MINER_PRIV_KEY
-      - DUMMY_PROVER=${DUMMY_PROVER:-1}
-      - ENABLE_FAUCET=1
+      - COORDINATOR_LISTEN=[::]:8000
+      - COORDINATOR_RPC_SERVER_NODES=server-testnet-geth:8545
+      - COORDINATOR_L2_RPC_URL=http://leader-testnet-geth:8545
+      - COORDINATOR_L1_RPC_URL=http://l1-testnet-geth:8545
+      - COORDINATOR_L1_BRIDGE=0x936a70c0b28532aa22240dce21f89a8399d6ac60
+      - COORDINATOR_L1_PRIV=$MINER_PRIV_KEY
+      - COORDINATOR_PROVER_RPCD_URL=http://dev:8001
```

Comment:
Can we move this back to the `useful env vars if running the proverd inside the dev image` .. 🙂

Reply:
sorry